experimental/desktop_photo_search/lib/src/widgets/data_tree.dart

```diff
@@ -0,0 +1,350 @@
+// Copyright 2019 The Flutter team. All rights reserved.
```

Comment:
Is this something you wrote in concert with Hans, or is it something he did for a different project that also works here?

Reply:
This code was lifted and adapted from a gist put together by @HansMuller. Split is from @DaveShuckerow.

Reply:
Hey @csells, the original idea for this sample was to showcase both `Split` and `DataTree`, but I fear the complexity of the `DataTree` in this sample is working against our larger goals of making Flutter on desktop approachable. Would you be ok with taking DataTree out and replacing it with something simpler? The upside is that we get a path to introducing shared preferences to store favorited photos.